verilog/tools/preprocessor/verilog_preprocessor.cc

```diff
@@ -128,13 +129,14 @@ static absl::Status MultipleCU(const SubcommandArgsRange& args, std::istream&,
                                std::ostream& message_stream) {
   // Parse the arguments into a FileList.
   std::vector<absl::string_view> cmdline_args(args.begin(), args.end());
-  auto parsed_file_list =
-      verilog::ParseSourceFileListFromCommandline(cmdline_args);
-  if (!parsed_file_list.ok()) {
-    return parsed_file_list.status();
+  verilog::FileList file_list;
+  if (auto status =
```

Comment:
I think there's a `RETURN_IF_ERROR` macro for this pattern, if you prefer.

Reply:
Neat, yes, that makes things more readable.